# rCMD : Command for the gripper.
#       0: release
#       1: grip
#       2: smart grip

uint16 rCMD

# rMGS : Magnet strength of the gripper.
#        0 ~ 100 [%]

uint16 rMGS